test/testkube/helm-testkube-values.yaml

```diff
@@ -110,11 +110,11 @@ mongodb:
   # Currently Bitnami doesn't support ARM: https://github.com/bitnami/charts/issues/7305
   image:
     # -- MongoDB image registry
-    registry: docker.io
+    registry: mcr.microsoft.com
```

Comment:
can you please update the readme about how and when to generate new the image in mcr? for e2e test excecution, are you adding section in readme too?